cmd/kubeadm/app/master/addons.go

```diff
@@ -221,11 +221,15 @@ func createKubeDNSPodSpec(cfg *kubeadmapi.MasterConfiguration) v1.PodSpec {
 				// that the DNS pod fails if the "nobody" _group_ doesn't exist. I think it's a typo in the Dockerfile manifest and
 				// that it should be "USER nobody:nogroup" instead of "USER nobody:nobody". However, this fixes the problem.
 				SecurityContext: &v1.SecurityContext{
-					RunAsUser: &dnsMasqMetricsUser,
+					RunAsUser: &dnsSidecarUser,
```

Comment:
I'd like to get rid of this workaround, but it's here due to what I think is a bug here: https://github.com/kubernetes/dns/blob/master/Dockerfile.sidecar#L21

The group nobody doesn't exist on most systems I think, at least it failed for me.
Can you just remove the whole line or at least fix it (might work with setting `nobody:nogroup` or something)